pkg/patch/patch.go

```diff
@@ -301,34 +301,40 @@ func patchWithContext(ctx context.Context, ch chan error, image, reportFile, pat
 	return eg.Wait()
 }
 
-func getOSType(ctx context.Context, osreleaseBytes []byte) (string, error) {
+func getOSType(ctx context.Context, osreleaseBytes []byte) (string, string, error) {
```

Comment:
since a lot of the work is repeated with getOSVersion, maybe we can combine the two functions into one and remove getOSVersion?

Reply:
sure, looking into it.